acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/tls/OkHttpClientHelpers.java

```diff
@@ -127,14 +127,14 @@ public static void populateFingerprintFile(
       final Path knownClientsPath, final TlsCertificateDefinition certDef)
       throws KeyStoreException, NoSuchAlgorithmException, CertificateException, IOException {
 
-      final List<X509Certificate> certs = getCertsFromPkcs12(certDef);
-      final StringBuilder fingerPrintsToAdd = new StringBuilder();
-      for (final X509Certificate cert : certs) {
-        final String fingerprint = generateFingerprint(cert);
-        fingerPrintsToAdd.append("localhost " + fingerprint + "\n");
-        fingerPrintsToAdd.append("127.0.0.1 " + fingerprint + "\n");
-      }
-      Files.writeString(knownClientsPath, fingerPrintsToAdd.toString());
+    final List<X509Certificate> certs = getCertsFromPkcs12(certDef);
+    final StringBuilder fingerPrintsToAdd = new StringBuilder();
+    for (final X509Certificate cert : certs) {
+      final String fingerprint = generateFingerprint(cert);
+      fingerPrintsToAdd.append("localhost " + fingerprint + "\n");
```

Comment:
nit: s/fingerPrint/fingerprint

Reply:
Mind if we don't make this change, given this file is not part of this PR otherwise?

Reply:
ok